exporter/splunkhecexporter/README.md

```diff
@@ -57,6 +57,9 @@ The following configuration options can also be configured:
 - `otel_to_hec_fields/severity_text` (default = `otel.log.severity.text`): Specifies the name of the field to map the severity text field of log events.
 - `otel_to_hec_fields/severity_number` (default = `otel.log.severity.number`): Specifies the name of the field to map the severity number field of log events.
 - `otel_to_hec_fields/name` (default = `"otel.log.name`): Specifies the name of the field to map the name field of log events.
+- `hec_heartbeat/hec_heartbeat_interval` (no default): Specifies the interval of sending hec heartbeat to the destination. If not specified, heartbeat is not enabled.
```

Comment:
Please add a descriptive section for this feature with examples. "observability_name" and "observability_dimensions" are relatively vague imo.